cms/djangoapps/models/settings/course_details.py

```diff
@@ -45,8 +45,8 @@ def __init__(self, org, course_id, run):
         self.enrollment_end = None
         self.syllabus = None  # a pdf file asset
         self.short_description = ""
-        self.overview = ""  # html to render as the overview
-        self.about_sidebar_html = "" # html to render as the about_sidebar_html
+        self.overview = ''  # html to render as the overview
```

Comment:
Commits are essentially the delta between 2 directory trees.

Whether to revert via `git revert` vs. "just changing the file". It's contextual and up to you. In this case, if you reverted the commit (e3ea86f), you'd also be reverting a lot of code that you actually want to keep. This is a big reason why separating changes into logical groups (PEP8 vs refactoring vs bug fix, etc.) is helpful; you have more fine grained control over how/what you revert. Here, it may be easier to just make the change manually.

Either way, we'll be squashing down these commits (rewriting the git history) into one or more logical set before we  merge (I'll walk you through this), so no worries for now.